go.mod

```diff
@@ -96,6 +95,7 @@ require (
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2 // indirect
 	github.com/prometheus/client_model v0.6.1 // indirect
+	github.com/prometheus/common v0.47.0 // indirect
```

Comment:
is this still required?

Reply:
it's pulled in as a dependency of a dependency, it's not directly needed anymore so shouldn't affect tests & the linter

Reply:
This will get updated as the prometheus exporter updates